doc/changelog.rst

```diff
@@ -10,6 +10,7 @@ unreleased
 
 - Add support for Python 3.7 up to 3.12.
 - Drop support for Python 3.6 and older.
+- Switch to implicit (native) namespace for ``sphinxcontrib``.
```

Comment:
Looks good, I would mark that as backwards incompatible in general. For or local version, it is fine.